client-admin/Dockerfile

```
@@ -8,8 +10,9 @@ RUN apk add git --no-cache
 COPY package*.json ./
 RUN npm install
 
-COPY polis.config.template.js polis.config.js
-# If polis.config.js exists on host, will override template here.
+COPY --from=config /config/config.js /app/config/
+COPY --from=config /config/*.yaml /app/config/
```

```suggestion
COPY --from=polis_config:latest /config/config.js /app/config/
COPY --from=polis_config:latest /config/*.yaml /app/config/
```

This was what I meant, though needed in other dockerfiles too